### PR TITLE
refactor: change structure, using actor model paradigm

### PR DESCRIPTION
**Pull request by ZigBalthazar**

### CODE REVIEW

File: sides/manager/errors.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package manager
+
+import (
+	"fmt"
+
+	bypassname "github.com/PACZone/teleport/types/bypass_name"
```
Comment (kehiy) [inline sides/manager/errors.go:6]: we changed it to bypassname

File: sides/manager/errors.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+type DupBypassError struct {
+	BypassName bypassname.BypassName
```
Comment (kehiy) [inline sides/manager/errors.go:10]: maybe change bypassname to bypass and add a name type to it? or include it on the message type module.

like: Message.bypassname?

File: sides/manager/manager.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		Bypasses: make(map[bypassname.BypassName]chan *message.Message, 10),
+	}
+}
+
+func (m *Mgr) Start() {
+	for {
+		select {
+		case msg := <-m.Highway:
+			err := m.routing(msg)
+			if err != nil {
```
Comment (kehiy) [inline sides/manager/manager.go:29]: add error handling here or continue! (maybe break an return error?)

**Review by kehiy — CHANGES_REQUESTED** [review]

File: sides/manager/errors.go (hunk 2 above)
Comment (ZigBalthazar) [reply sides/manager/errors.go:10]: bypassname changed to bypass, now, Names is a type of it

File: sides/manager/manager.go (hunk 3 above)
Comment (ZigBalthazar) [reply sides/manager/manager.go:29]: if err will not be nil, continue for now, I think we should review it again later

File: Makefile [hunk 4]
```diff
@@ -20,7 +20,7 @@ build:
 
 ### ABIs (EVM contracts)
 build-abis:
-	abigen --abi ./abis/WrappedPac.json --pkg polygonClient --type WrappedPac --out ./client/polygon_client/wrapped_pac.go
+	abigen --abi ./abis/WrappedPac.json --pkg polygonClient --type WrappedPac --out ./sides/polygon/wrapped_pac.go
```
Comment (kehiy) [inline Makefile:23]: update package name

File: types/bypass/bypass_name.go [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package bypass
+
+type (
+	Names string
```
Comment (kehiy) [inline types/bypass/bypass_name.go:4]: consider using `name`

**Pull request merged**